eslint.config.mjs

```diff
@@ -465,7 +465,10 @@ export default [
       }],
       'eslint-plugin-js/dot-location': ['warn', 'property'],
       'eslint-plugin-js/eol-last': ['warn', 'always'],
-      'eslint-plugin-js/func-call-spacing': ['warn', 'never'],
+      'eslint-plugin-js/function-call-argument-newline': [
```

Comment:
Most of the changes in this commit don't seem more readable to me, TBH.

Also, I don't see how

```js
foo('bar',
    'baz);
```

is much different from

```js
foo={'bar' +
     'baz'}
```

which is allowed by our JSX rules?

Reply:
I guess the only real case where I liked them on the same line are the mutate-cow property paths. Though they're not that bad on separate lines either, so I guess the rule is worth having overall.